RickLeinecker/Fortuna
# Finished code for all 10 weapons and their respective bullets

**Description**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves #197 
Resolves #198

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
You can change the weapons tanks are given in the TankLoader class to see how different things work. Here is a demo of the missiles vs the Lancer:
![LancervsMissiles2](https://user-images.githubusercontent.com/18317476/77470978-ed0ac380-6de7-11ea-8df2-e7eec53055c8.gif)


**Final Checklist:**
Go down the list and check them off.
